lib/services/local/headlessauthn_watcher.go

```diff
@@ -84,14 +77,16 @@ func (cfg *HeadlessAuthenticationWatcherConfig) CheckAndSetDefaults() error {
 	return nil
 }
 
-// HeadlessAuthenticationWatcher is a custom backend watcher for the headless authentication resource.
+// HeadlessAuthenticationWatcher is a light weight backend watcher for the headless authentication resource.
 type HeadlessAuthenticationWatcher struct {
+	// HeadlessAuthenticationWatcherConfig is the configuration of a HeadlessAuthenticationWatcher.
```

Comment:
We can probably do without this comment. I don't think it says anything that's not already obvious.